packages/round-manager/src/features/round/RoundApplicationForm.tsx

```diff
@@ -304,32 +307,21 @@ export function RoundApplicationForm(props: {
   };
 
   const formSubmitModals = () => (
-    <InfoModal
```

Comment:
@vacekj rather than completely removing it, i think its better we make this conditional as well depending on useNewRoundInit?

Reply:
all the chains will use the new UI soon, useNewRoundInit will be removed and all chains will use the aio deploy soon. So no reason to keep it.